definitions/json/AuthorisationCaseField.json

```diff
@@ -1835,18 +1619,6 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
-  {
-    "CaseTypeId": "ET_EnglandWales",
-    "CaseFieldID": "claimant",
```

Comment:
should be kept to know who the claim is against. also field is referenced in ET3 journey and can break it